core/lib/generators/spree/install/install_generator.rb

```diff
@@ -75,6 +75,14 @@ def create_overrides_directory
       empty_directory "app/overrides"
     end
 
+    def copy_views
+      if defined? Spree::Frontend && !options[:quiet]
```

Comment:
@bbonislawski please use `frontend_available?` from:

https://github.com/spree/spree/blob/a9f7ad3aa14972f1ca057e5aec7d5f6aa409e4eb/core/lib/spree/core/components.rb#L12

Reply:
I didn't know that something like that exists. Thanks!